speech/api/speech_async_grpc.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
+# Copyright (C) 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Sample that transcribes a FLAC audio file stored in Google Cloud Storage,
+using async GRPC."""
+
+import argparse
+import time
+
+from gcloud.credentials import get_credentials
+from google.cloud.speech.v1beta1 import cloud_speech_pb2 as cloud_speech
```

Comment:
In general I don't recommend aliasing imports. Is this a standard grpc practice?